lib/exceptions/backends/log_result.rb

```diff
@@ -12,8 +12,9 @@ def initialize(backend, logger = ::Logger.new(STDOUT))
         @logger  = logger
       end
 
-      def notify(exception, options = {})
-        backend.notify(exception, options).tap do |result|
+      def notify(*args)
+        backend.notify(*args).tap do |result|
```

Comment:
hmmmm, i figured it out at one point a few weeks ago. let me try and repro. i had to do the following fix in r101-api before:

```ruby
# XXX: monkeypatch exceptions to stop mangling args before sending them to
# backends. if rollbar eval is successful, let's merge this into exceptions
# for reals.
Exceptions.module_eval do
  def self.notify(*args)
    backend.notify(*args)
  end
end
```

i think i have to do also do this a level down. let me see if i can get an isoated repro highlighting the issue.